kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -40,6 +40,8 @@ define([
     UUID
 ) => {
     'use strict';
+    const cssClassName = 'kb-staging-table';
```

Comment:
Changes in this file:
- switch out `kb-staging-table` with `cssClassName`
- a lot of the font awesome icons have an extra span around them; I have removed the surplus HTML
- fixed some ESLint errors
- BEM-ified element names